### PR TITLE
refactor: move social asset into page meta

### PR DESCRIPTION
**Pull request by TrySound**
Removed social asset form top level and moved into generated page meta.
Will be less generated data and simple page head.

### CODE REVIEW

File: packages/sdk/src/page-meta-generator.test.ts [hunk 1]
```diff
@@ -70,6 +72,22 @@ test("generate complete static page meta factory", () => {
...
+          {
+            id: "social-image-id",
+            type: "image",
+            format: "",
+            projectId: "",
+            size: 0,
+            name: "social-image-name",
+            description: null,
+            createdAt: "",
+            meta: { width: 0, height: 0 },
```
Comment (kof) [inline packages/sdk/src/page-meta-generator.test.ts:87]: intersting defaults

**Review by kof — APPROVED** [review]
didn't test

File: packages/sdk/src/page-meta-generator.test.ts (hunk 1 above)
Comment (TrySound) [reply packages/sdk/src/page-meta-generator.test.ts:87]: just a stub

Comment (kof) [reply packages/sdk/src/page-meta-generator.test.ts:87]: ah its a test, sorry

**Pull request merged**